install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,44 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access a
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
 
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Microsoft Azure Disk as a persistent volume on the {product-title} version 3.5 or later, you must enable Azure Cloud Provider.
+* All virtual machines (VM) configured as OpenShift nodes, must belong to a single
```

Comment:
Consider; 

* All OpenShift node virtual machines (VMs) running in Microsoft Azure must belong to a single resource group.